### PR TITLE
replaces Resources with Tracked Async Data.

### PR DESCRIPTION
**Pull request by stopfstedt**
refs https://github.com/ilios/ilios/issues/5371

### CODE REVIEW

File: packages/ilios-common/addon/components/learnergroup-tree.js [hunk 1]
```diff
@@ -46,21 +37,6 @@ export default class LearnergroupTree extends Component {
     return !this.filterMatch;
   }
 
-  /**
-   * Recursively search a group tree to see if there are any children which have not been selected.
-   **/
-  async getHasUnSelectedChildren(children, selectedGroups) {
```
Comment (jrjohnson) [inline packages/ilios-common/addon/components/learnergroup-tree.js:52]: I'm assuming this is the dead code? Nice find!

**Review by jrjohnson — APPROVED** [review]
Never did get used to it, like to see it go.

**Comment by jrjohnson** [comment]
@michaelchadwick I tossed this over to you for a look, but I mainly just want to make sure you take a look at [this post](https://www.pzuraq.com/blog/introducing-use) and as a reminder for us to talk about the evolution of async in ember and how it's sprinkled into Ilios next time we have some in person time.

**Comment by michaelchadwick** [comment]
@jrjohnson I want to be up front and say that I spent a non-trivial time just clicking the light/dark mode toggle on that blog because the transition is so smooth and the little house's details change in a cool way :D

Still getting through the blog post, but will report back on my thoughts.

**Comment by michaelchadwick** [comment]
@jrjohnson Got through the blog post. The whole paradigm shift it seemed to introduce seemed a lot like VueJS (JS + HTML in same file). Also, I see Ilios uses `ember-could-get-used-to-this` and the `@use` decorator quite a lot, but this change seems to be moving Ilios _away_ from that to `TrackedAsyncData`. I feel like I don't have enough background to say whether this is better or worse, so I'll trust the team..._for now_. ;P

**Pull request merged**